.github/maintainers/maintainers.go

```diff
@@ -0,0 +1,155 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+	"text/template"
+
+	"github.com/bradleyfalzon/ghinstallation/v2"
+	"github.com/google/go-github/v69/github"
+)
+
+type TeamMember struct {
+	Login     string `json:"login"`
+	AvatarURL string `json:"avatar_url"`
+	HTMLURL   string `json:"html_url"`
+}
+
+func main() {
+	tr, err := ghinstallation.New(
+		http.DefaultTransport,
+		1185968, 63003777,
```

Comment:
I prefer to avoid undocumented magic numbers. Either add a comment explaining what these are for, or define constants that name them.